languages.json

```diff
@@ -773,18 +778,11 @@
       "line_comment": ["#"],
       "extensions": ["nix"]
     },
-    "ObjectiveC": {
+    "ObjectiveCCpp": {
       "name": "Objective-C",
       "line_comment": ["//"],
       "multi_line_comments": [["/*", "*/"]],
       "quotes": [["\\\"", "\\\""]],
-      "extensions": ["m"]
```

Comment:
You shouldn’t remove another language. I would prefer if matlab didn’t include the `m` extension.

Reply:
Reality is that `.m` is by far the most common extension for existing Matlab code. This is the logic used by `cloc` to disambiguate.
https://github.com/AlDanial/cloc/blob/master/Unix/cloc#L10870-L10993